datamodel_code_generator/__init__.py

```diff
@@ -500,15 +500,18 @@ def get_header_and_first_line(csv_file: IO[str]) -> Dict[str, Any]:
         header += f'\n#   version:   {get_version()}'
 
     file: Optional[IO[Any]]
-    for path, body_and_filename in modules.items():
-        body, filename = body_and_filename
+    for path, (body, filename) in modules.items():
```

Comment:
Oh, I didn't realize the way to unpack the tuple in the `for` statement.
Thank you!!